tests/extension/extension.test.ts

```diff
@@ -74,6 +74,8 @@ vi.mock('../../src/extension/grpc/runner/v1', () => ({}))
 test('initializes all providers', async () => {
   const configValues = {
     binaryPath: 'bin',
+    // This is needed for the AIManager to initialize.
+    'runme.aiBaseURL': 'http://localhost:8080/api',
```

Comment:
I'd suggest using a different port, `8080`, which is over-used. Fine inside container images, but running locally, there's a high likelihood another process bound the port locally.

Reply:
Good Idea. I changed the default to 8877.
WDYT?

Reply:
Good choice, @jlewi!